dev-lang/retro/retro-2021.4.recipe

```diff
@@ -0,0 +1,69 @@
+SUMMARY="Clean, elegant, and pragmatic dialect of Forth"
+DESCRIPTION="Retro is a clean, elegant, and pragmatic dialect of Forth. \
+It provides a simple alternative for those willing to make a break from legacy \
+systems.
+
+The language draws influences from many sources including traditional \
+Forth systems, cmForth, colorForth, Factor, and Parable. It was \
+designed to be easy to grasp and adapt to specific uses.
+
+The basic language is very portable. It runs on a tiny virtual \
+machine (Nga), which is written in C. There are multiple interface \
+options, the main one (rre) is buildable with just the standard C \
+compiler and libraries on most systems (tested at various points \
+on Linux, NetBSD, macOS, and Windows, on x86, x86-64, PPC [emulated], \
+and various ARM processors)."
+HOMEPAGE="http://forthworks.com/retro/"
+COPYRIGHT="2008-2021, Charles Childers"
+LICENSE="ISC"
+REVISION="1"
+SOURCE_URI="http://forthworks.com/retro/r/RETRO12-$portVersion.tar.gz"
+CHECKSUM_SHA256="9d43aae72527e7b18fdbd3b4237d425b79ad95bb3d0dee6655264970478524b0"
+SOURCE_DIR="RETRO12-$portVersion"
+
+ARCHITECTURES="x86_64 ?x86 ?x86_gcc2"
```

Comment:
I gather you don't have a chance to checkout for x86_gcc2 build, I'll check later, but if it doesn't build for gcc2 it will need to be rebuild for gcc8 and needs a SECONDARY_ARCHITECTURES with $secondaryArchSuffix in places were need be

Reply:
Did a build and indeed needs a secondary architecture, please adjust (if you have any problems with it just call for help) :)

Reply:
x86_64 should come at the end, x86_gcc2 at the beginning.

Reply:
Will do, but may I ask what is the reason for it? Can I learn somewhere about those haikuporter pecularities?

Reply:
fixed